app/controllers/admin/statuses_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Admin
+  class StatusesController < AdminController
```

Comment:
I noticed that PaymentsController ShipmentsController are doing basically the same thing with different resources I created this base controller and all others need only to set up their own private resource method. Also updating order status is the same thing so I decided to create a separate controller OrderStatusesController co it can safely inherit from StatusesController.

Reply:
Very nice refactor!